packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -198,6 +200,15 @@ public void onMethodCall(MethodCall call, final Result result) {
     }
   }
 
+  @SuppressWarnings("ConstantConditions")
```

Comment:
Why is this being suppressed?

Reply:
The casting of `Exception` to `RuntimeException`. From my understanding of Java, `CameraAccessException` is the only checked exception that will come from these methods. Otherwise it should be a `RuntimeException`.